### DISCUSSION TITLE
Project Update June 2023

### DISCUSSION THREAD

**Discussion by DeeEmm** [DIY-Flow-Bench/DIY-Flow-Bench#153] [Announcements]
So another new month has snuck up. 

There are no current changes to the code although I have added an additional pin definition for the ESP32-WROOM-32 board which will be in the next code release, whenever that is. The WROOM 32 boards are generally the small footprint ESP32's commonly found on IOT websites. Whilst they cannot be used with the official shield, they are of course functionally identical and are a cheap way to DIY a solution.

Current project status is still in the validation stage, so if you are considering building a flow bench it would be good for the project if you build and test with the DIYFB code.

I don't foresee any code updates this month as I am tied up with other stuff, this will hopefully give some time for those with partially completed builds to get up and running.

As always, if you get stuck or have any questions, raise a topic in the forums and I will endeavour to assist

**Comment by black-top** [comment]
I have received the pcb board, but I did not assembled it because some components are still missing, but the main problem my work stopped for a while because I badly injured my hand. So there is little I can do for a while.  Will be very interesting to hear the results of your measurements.

**Reply by DeeEmm** [reply]
Sorry to hear about your hand. I hope you have a speedy recovery. What components do you need?

**Reply by black-top** [reply]
I almost recovered from the injury, so I will be back to flowbench work soon. I need a level shifter board because it got lost in the mail, but I think I will use a different board that is working fine in my current setup. Also, I need a different baro sensor. If I remember correctly I ordered the 5V version because in the circuit it was connected to 5V but actually a 3.3V version was required, and I need to decide what main pressure sensor to use because my current 10KPA is easily overloaded without pressure control, but I'm not sure if I need more resolution because maf data is quite consistent with different pressures, so I probably will  do more testing with my current old setup to see how to improve measurements before ordering more parts and finishing new setup with pcb.  Any news or updates since I disappeared?

**Reply by DeeEmm** [reply]
Thats great news. Happy to hear that your injury is healing.

Nothing has changed since you last visited. At the moment all of the prototype PCBs have now been sent out so I've mostly been waiting for people to assemble them and start collecting data so that I have some results to compare. I am apprehensive to make major changes without validating them from multiple data sources. I've also been tied up with real life stuff so have not had lots of time to focus on this project.
